tests/testthat/test-consecutive-id.R

```diff
@@ -0,0 +1,20 @@
+test_that("works with simple vectors", {
+  expect_equal(consecutive_id(c(1, 1, 2, 1, 2)), c(1, 1, 2, 3, 4))
+})
+
+test_that("handles data frames", {
+  df <- tibble(x = c(1, 1, 1), y = c(1, 2, 1))
+  expect_equal(consecutive_id(df), c(1, 2, 3))
```

Comment:
```suggestion
  df <- tibble(x = c(1, 1, 1, 1), y = c(1, 2, 2, 1))
  expect_equal(consecutive_id(df), c(1, 2, 2, 3))
```

This seems slightly more informative for this function? i.e. it now tests:
- Identical consecutive rows get the same id (now tested with my suggestion)
- A new row always increments the id (was already tested)